hack/common/Makefile

```diff
@@ -97,6 +104,10 @@ apply-serverless: ## Apply the k3d serverless CR.
 
 ### Internal Dependencies
 
+.PHONY: kyma
+kyma: ## Install kyma-cli under the `bin` folder if it does not exist.
```

Comment:
What whith this comment? Now `kyma` will be described in previous section `Module managenment`.